scripts/pocs-shell.py

```diff
@@ -190,7 +190,17 @@ def do_reset_pocs(self, *arg):
 
         Does NOT park the mount, nor execute power_down.
         """
-        self.pocs = None
+        if self.pocs.observatory.mount.is_parked is False:
+            print("ATTENTION: The mount is not in the parked position and reset_pocs will not move the mount to park position!")
```

Comment:
There are some convenience functions at the bottom that can print in different colors.

```suggestion
            print_warning("ATTENTION: The mount is not in the parked position and reset_pocs will not move the mount to park position!")
```